content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+  ([Issue #32539](https://github.com/istio/istio/issues/32539))
+
+- **Fixed** a bug where multiple gateways on the same port with `SIMPLE` and `PASSTHROUGH` modes is not working correctly.  ([Issue #33405](https://github.com/istio/istio/issues/33405))
+
+- **Fixed** a bug where Istio config gen fails when sum of endpoint weights over uint32 max.  ([Issue #33536](https://github.com/istio/istio/issues/33536))
+
+- **Fixed** smart DNS support in Istio CNI.
+  ([Issue #29511](https://github.com/istio/istio/issues/29511))
+
+- **Fixed** a bug in Kubernetes Ingress causing paths with prefixes of the form `/foo` to
+match the route `/foo/` but not the route `/foo`.
+  
+
+- **Fixed** an issue allowing ServiceEntries to act as instances in other namespaces.
+  
+
+- **Fixed** an issue causing proxies to send `Transfer-Encoding` headers with `1xx` and `204` responses.
+  
+
+- **Fixed** reconciliation logic in the validation webhook controller to rate-limit
+the retries in the loop. This should drastically reduce churn (and generated logs)
+in cases of misconfiguration.
+  ([Issue #32210](https://github.com/istio/istio/issues/32210))
+
+
+
+- **Optimized** generated routing configuration to merge virtual hosts with the same routing configuration. This improves performance for Virtual Services with multiple hostnames defined.
+  ([Issue #28659](https://github.com/istio/istio/issues/28659))
+
+
+## Security
+
+
+
+
+
+
+
+
+
+
+
+## Telemetry
+
+
+
+- **Updated** Prometheus telemetry behavior for inbound traffic to disable host header fallback by default. This will
+prevent traffic coming from out-of-mesh locations from potential polluting the `destination_service` dimension in
+metrics with junk data (and exploding metrics cardinality). With this change, it is possible that users relying on
+host headers for labeling the destination service for inbound traffic from out-of-mesh workloads will see that traffic
+labeled as `unknown`. The behavior can be restored by modifying Istio configuration to remove the `disable_host_header_fallback: true`
+configuration.
+  
+
+- **Added** support for [Apache SkyWalking](https://skywalking.apache.org/) tracer. Now you can run the `istioctl dashboard skywalking` command to view skywalking dashboard UI.
+  ([Issue #32588](https://github.com/istio/istio/pull/32588))
+
+- **Added** a new metric to `istiod` to report server uptime.
+  
+
+- **Added** a new metric (`istiod_managed_clusters`) to `istiod` to track the number of clusters managed by an
+`istiod` instance.
+  
+
+
+- **Fixed** Prometheus [metrics merging](/docs/ops/integrations/prometheus/#option-1-metrics-merging) to
+correctly handle the case where the application metrics are exposed as [OpenMetrics](https://github.com/OpenObservability/OpenMetrics).
+  ([Issue #33474](https://github.com/istio/istio/issues/33474))
+
+
+
+
+
+## Extensibility
+
```

Comment:
@istio/wg-policies-and-telemetry-maintainers anything worth mentioning here?

Reply:
@myidpt @shankgan Anything to add here? Supporting multiple roots?

@yangminzhu @lizan Anything to add to the release notes?

Reply:
@istio/wg-policies-and-telemetry-maintainers Bump

Reply:
I cannot think of anything. @kyessenov @mandarjog @dgn @mathetake Can you think of anything? If not, we probably could remove this section.

Reply:
nothing from my side